### PR TITLE
Feature#3/sign up screen

### PR DESCRIPTION
**Pull request by ljau**
# Description
Created sign up screen view
 

  ## Screenshots 
image

### CODE REVIEW

File: GoToProject/src/containers/SignUp/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import {RecordHeader} from '../../components/RecordHeader';
+import {
+
+  Layout,
+} from '../../components/styled';
```
Comment (OswaldoPineda) [inline GoToProject/src/containers/SignUp/index.js:6]: you can refactor this in one line

File: GoToProject/src/containers/SignUp/index.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import { CustomInput } from '../../components/customInput';
+import { ButtonCustom } from '../../components/ButtonCustom';
+
+export const SignUp = () => {
+
+  return (
+
```
Comment (OswaldoPineda) [inline GoToProject/src/containers/SignUp/index.js:13]: remove this extra space

**Review by OswaldoPineda — DISMISSED** [review]

File: GoToProject/src/components/styled.js [hunk 3]
```diff
@@ -57,7 +57,7 @@ export const TextButton = styled.Text`
 export const InputText = styled.TextInput`
   background: ${colors.darkGray};
   width: 100%;
-  height: ${({h}) => h ?? '35%'};
+  height: ${({h}) => h ?? '55px'};
```
Comment (LeyaimMagma) [inline GoToProject/src/components/styled.js:60]: change the variable name `h`  could be confused

File: GoToProject/src/containers/SignUp/index.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+import { ButtonCustom } from '../../components/ButtonCustom';
+
+export const SignUp = () => {
+
+  return (   
+        <Layout>
+          <RecordHeader title="Sign Up" />
+          <CustomInput label="Full Name" />
+          <CustomInput label="Email" />
+          <CustomInput label="Password" />
```
Comment (LeyaimMagma) [inline GoToProject/src/containers/SignUp/index.js:14]: could you refactor this?

File: GoToProject/src/components/CustomInput/index.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import {FlexContainer, InputLabel, InputText} from '../styled';
+
+export const CustomInput = ({...props}) => {
+  return (
+    <FlexContainer h="15%" alin>
```
Comment (LeyaimMagma) [inline GoToProject/src/components/CustomInput/index.js:6]: bad prop name

**Review by LeyaimMagma — CHANGES_REQUESTED** [review]

**Review by YilKanda — COMMENTED** [review]
When you construct new variables, remember to use representative names of what they do, to avoid future confusion (either to yourself or to other team members).

File: GoToProject/src/containers/SignUp/index.js (hunk 4 above)
Comment (ljau) [reply GoToProject/src/containers/SignUp/index.js:14]: any suggestion? I already reduced it from writting 3 times the input box and label code (custom input component includes the input and the label), to just write it once and call each time needed, for this screen we need 3 inputs.

Comment (LeyaimMagma) [reply GoToProject/src/containers/SignUp/index.js:14]: you can use a map() or section/flat list

Comment (LeyaimMagma) [reply GoToProject/src/containers/SignUp/index.js:14]: <CustomInput label="{input name}" action=(action) />

const login = [
  {
    title: 'Full name',
   action: () => {action()}
  },
  {
    title: 'email',
  },
  {
    title: 'password',
  },
];

Comment (ljau) [reply GoToProject/src/containers/SignUp/index.js]: Ya actualice el archivo, y ahora utilizo map() para mostrar el componente que se repetia.

Comment (ljau) [reply GoToProject/src/containers/SignUp/index.js]: I have updated the file and now I use map() to display the component without repeating the code.

**Review by LeyaimMagma — APPROVED** [review]

**Pull request merged**